lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -172,7 +172,8 @@ <h3 class="hd hd-3">${_("Course Team Management")}</h3>
         ${_("Course team members with the Staff role help you manage your course. "
```

Comment:
I see here we're only updating:
Staff
Limited Staff
Course Admin
Beta tester
And the discussion roles will be changed here: https://github.com/openedx/edx-platform/pull/32436/, do those include the rest of the roles?
Discussion Admins
Discussion moderators
Group community TA
Community TA
Course Data Researcher -> This doesn't mention the users' enrollment status

Reply:
@mariajgrimaldi That is correct.
`Course Data Researcher`: Is also handled by the same API, so I updated the text to include info about auto-enrollment.